src/compiler/reviews.md

```diff
@@ -1,104 +1,81 @@
 
 It is the purpose of code reviews to:
 
-- **Reduce** the risk of introducing **bugs** and usability and performance
-  **regressions**.
-- Keep our code **maintainable**: **readable**, **documented** and
-  **well-tested**.
-- Ensure that changes are made with the big picture and **appropriate context in
-  mind**. This is particularly relevant for changes that seem harmless in
-  isolation but are problematic or undesirable in the larger context.
-
-Reviewing accomplishes this by bringing in another set of eyes to look at a
-proposed change from a different perspective, which increases the chance of
-catching mistakes early and uncovering potential blind spots in the reasoning
-behind the change.
+- **Reduce** the risk of introducing **bugs** and usability and performance **regressions**.
+- Keep our code **maintainable**: **readable**, **documented** and **well-tested**.
+- Ensure that changes are made with the big picture and **appropriate context in mind**. This is
+  particularly relevant for changes that seem harmless in isolation but are problematic or
+  undesirable in the larger context.
 
+Reviewing accomplishes this by bringing in another set of eyes to look at a proposed change from a
+different perspective, which increases the chance of catching mistakes early and uncovering
+potential blind spots in the reasoning behind the change.
 
 ## Basic Reviewing Requirements
-
 There are a number of requirements that need to be met in order for reviewing to
 be effective:
 
 - Reviewers must have a sufficient **understanding of the code** under review.
-  - This is important to help spot non-obvious, unintentional side effects of a
-    given change.
+  - This is important to help spot non-obvious, unintentional side effects of a given change.
 - Pull request authors must provide:
-  1. A concise high-level **description of the change** and (2) the
-     **rationale** behind it, unless the change is extremely trivial, like typo
-     fixes.
-  2. For the rationale to be even more useful, authors are encouraged to list
-     potential **points of contention**, **compromises** that needed to be made,
-     **alternative approaches** that have been considered, **relevant
-     documentation, discussions and context**, etc.
-    - Reviewing code is difficult, and reviewers only have a **limited amount of
-      time** to do it. Jump-starting the review process by not making the
-      reviewer puzzle together the intention and context of a pull request will
-      not only speed things up but also improve the quality of the review.
-- Reviewers must have a good idea on whether they are the **right person to
-  approve** the change.
-  - Knowledge of the code under review is an obvious but not the only criteria
-    for answering this question.
+  1. A concise high-level **description of the change** and (2) the **rationale** behind it,
+     unless the change is extremely trivial, like typo fixes.
```

Comment:
Suggestion:

```suggestion
  1. A concise high-level **description of the change** and (2) the **rationale** behind it,
     unless the change is extremely trivial.
```

(recent wave of typo fixes...)

Reply:
I've kept this because even if we sometimes discourage typo fixes, it is a valid case where you don't need a big description.